terraform-incubator/access-the-data/versions.tf

```diff
@@ -0,0 +1,42 @@
+// Get configuration from the shared infrastructure
```

Comment:
Projects wanting to copy from AtD should be able to copy this file verbatim. There's some boilerplate that should be the same for every project.